README.md

```diff
@@ -1,3 +1,34 @@
 # Augur
 
-This repository will serve as the central location for Augur Issues and other items.
+## Define: Augur Protocol
+Augur is a decentralized oracle and peer to peer protocol for prediction markets. Augur is free, public, open source software, portions of which are licensed under the General Public License (GPL) and portions of which are licensed under the Massachusetts Institute of Technology (MIT) license. Augur is a set of smart contracts written in Solidity that can be deployed to the Ethereum blockchain.
+
+Augur is a protocol, freely available for anyone to use however they please. Augur is accessible through a desktop client app, similar to interacting with an Ethereum or Bitcoin node. Users of the Augur protocol must themselves ensure that the actions they are performing are compliant with the laws in all applicable jurisdictions and must acknowledge that others’ use of the Augur protocol may not be compliant. Users of the Augur protocol do so at their own risk.
+
+For more information about the Augur protocol and the Forecast Foundation, [check out the FAQ](https://www.augur.net/faq/).
+
+## Want to try Augur?
+
+[Grab the latest release of Augur App](https://github.com/AugurProject/augur-app/releases) to connect to the Ethereum Network and explore what the Augur Community has done.
+
+## 100% Fully Open Source
+
+You can view, edit, and contribute to Augur via our 100% open source repositories hosted on GitHub!
```

Comment:
Awkward wording, suggest maybe "All of Augur's code is hosted in open source repositories on GitHub.", and removing the word "Fully" from the section header?